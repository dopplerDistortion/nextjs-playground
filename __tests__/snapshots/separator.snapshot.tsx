import {Separator} from '@radix-ui/react-separator';
import {render} from '@testing-library/react';

const DummyLabel = () => <Separator />;
it('renders separator unchanged', () => {
  const {container} = render(<DummyLabel />);
  expect(container).toMatchSnapshot();
});
