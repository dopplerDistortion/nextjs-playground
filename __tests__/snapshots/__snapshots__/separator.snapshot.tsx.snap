// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders separator unchanged 1`] = `
<div>
  <div
    data-orientation="horizontal"
    role="separator"
  />
</div>
`;
